pimd/pim6_main.c

```diff
@@ -168,12 +168,12 @@ int main(int argc, char **argv, char **envp)
 	 */
 	pim_error_init();
 	pim_vrf_init();
-#if 0
```

Comment:
The correct fix during the release freeze is to remove `VTYSH_PIM6D` from `#define VTYSH_RMAP` in `vtysh/vtysh.h`, since noone has even tried the route-map code in pim6d.

Reply:
(also `VTYSH_ACL`) i guess